configs/config.d/forward.conf

```diff
@@ -0,0 +1,6 @@
+<source>
+  @type forward
+  port 24224
```

Comment:
I thought it did make sense - we are asking users to edit the files in that
directory as necessary, and including the default in the file makes it
easier to see right there what the default is, and to decide whether it
needs to be changed. Otherwise, users will have to find the documentation
that's in a totally different spot.

On Thu, 2 Jun 2016, 18:57 igorpeshansky, <notifications@github.com> wrote:

> In configs/config.d/forward.conf
> <https://github.com/GoogleCloudPlatform/fluentd-catch-all-config/pull/7#discussion_r65577922>
> :
>
> > @@ -0,0 +1,6 @@
> > +<source>
> > +  @type forward
> > +  port 24224
>
> This is the default, isn't it? Does it still make sense to set it
> explicitly?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/fluentd-catch-all-config/pull/7/files/d87ae65a65f4452413bbbbacd3f0b242fba327c2#r65577922>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AALxtGnuVR7pHv2o0qXxrkJuK0dO1Yb0ks5qHwuUgaJpZM4Iez4l>
> .
>